crm_request/models/request.py

```diff
@@ -17,8 +17,9 @@ class CrmClaim(models.Model):
         help="The destination email address that the contacts used.")
     code = fields.Char(string='Number')
     claim_type = fields.Many2one(string='Type')
-    user_id = fields.Many2one(string='Assign to')
-    stage_id = fields.Many2one(group_expand='_read_group_stage_ids')
+    user_id = fields.Many2one(string='Assign to', track_visibility=False)
```

Comment:
Why did you remove the track_visibility attributes? It's a bit strange to remove this, as it won't notify anyone if we change the assignee directly from the field. Maybe then you should put the field readonly in the view to make sure nobody changes it without going through your wizard.